cfgov/unprocessed/js/routes/on-demand/expandable-group.js

```diff
@@ -20,7 +20,7 @@ for ( var i = 0, len = expandables.length; i < len; i++ ) {
   expandable.init();
 }
 
-selector = '.expandable-group-prototypes .o-expandable-group';
+selector = '.o-expandable-group';
```

Comment:
As noted in chat, this'll fall apart if an expandable and an expandable group are both added to a page, so scoping of the expandable within the expandable group will need to be worked out at some point. OK for this PR though.